RF_ASM/Utils.cs

```diff
@@ -52,5 +52,13 @@ public static void DisplayHexDump(byte[] data)
                 Console.WriteLine();
             }
         }
+
+        public static void PadBytes(int desiredLength, byte[] ba)
+        {
+            while (ba.Length < desiredLength)
```

Comment:
Since prepend does nothing, this creates an infinite loop if you gave the function appropriate inputs.

Reply:
The function has zero references... why did you make 3 conversations for this